test/unit/dom/SVGElement.js

```diff
@@ -0,0 +1,46 @@
+import canPatchNativeAccessors from '../../../src/util/can-patch-native-accessors';
+import create from '../../lib/create';
+
+describe('dom: SVGElement', function () {
+  runTests('div');
+  runTests('slot');
+  runTests('host');
+  runTests('root');
+
+  function runTests(type) {
+    describe(`${type}: `, () => {
+      let host, root, slot, div, elem;
```

Comment:
Unfortunately no, slot, host and root go together, one cannot exist without the others.